### DISCUSSION TITLE
Kaspersky Compliance

### DISCUSSION THREAD

**Discussion by rustdesk** [rustdesk/rustdesk#5624] [Team]
# Resolved
- [x] It can be configured with "Hide connection management window" option, which disables all notifications and indication that someone is interacting with the system. Also in this case, user doesn't have any interface to interrupt connection.
> https://github.com/rustdesk/rustdesk/pull/5544
- [x] there is no indication of file operations
> https://github.com/rustdesk/rustdesk/pull/5623
- [x] the "silent" cli-installation shows no notification to the end user after the process is complete.

> spawn a windows dialog pop after installation done, (do not block current process). https://github.com/rustdesk/rustdesk/pull/5632

- [x] tray icon displays no information about the active session

> show number of connections as tooltip as AnyDesk
https://github.com/rustdesk/rustdesk/pull/5654

- [x] the process responsible for showing the tray icon is independent from the process responsible for the session, therefore it's possible to remove the tray icon by terminating the corresponding process while at the same time keeping the active remote session

> check if tray exists for new connection, if not, spawn tray icon, following anydesk behavior.
https://github.com/rustdesk/rustdesk/pull/5654

- [x] the launch arguments "--cm-no-ui" "--cm --hide" allows the software to work silently

> disable this for Windows, they are not used in Windows.
https://github.com/rustdesk/rustdesk/pull/5654

**Comment by rustdesk** [comment]
@21pages

**Comment by quyvsquy** [comment]
>     * [x]  It can be configured with "Hide connection management window" option, which disables all notifications and indication that someone is interacting with the system. Also in this case, user doesn't have any interface to interrupt connection.

I disagree with the assertion that the "Hide connection management window" option cannot be controlled by the user and that it disables all notifications and indications of someone interacting with the system. In most cases, this feature is configurable by the user, allowing them to choose whether to enable or disable it. However, it's important to emphasize the need for caution and a full understanding of the associated security risks when deciding to hide these notifications. While users may be the masters of their machines, responsible usage and awareness of potential security implications are still paramount.
I think that feature should remain available to users.
@rustdesk

**Reply by rustdesk** [reply]
You do not need to convince us, we do not have choice. You need to convince Kaspersky.

**Comment by rustdesk** [comment]
- [x] the operations of deleting/creating folders and deleting files are not displayed
https://github.com/rustdesk/rustdesk/pull/6303
- [x] the "--cm --hide" arguments still create a connection process with a hidden window that closes after a few seconds. It's still possible to establish a hidden remote connection within this time window. 
https://github.com/rustdesk/rustdesk/pull/6333
- [x] the CM window process can be terminated while keeping the remote session remains active, similar to the tray icon process
https://github.com/rustdesk/rustdesk/pull/6333
- [x] the "Block user input" feature cannot be disabled on the remote computer side
https://github.com/rustdesk/rustdesk/pull/6303
